.editorconfig

```diff
@@ -1,5 +1,16 @@
 root = true
 
-[*.js]
-indent_style=tab
-trim_trailing_whitespace=true
\ No newline at end of file
+[*]
+indent_style = tab
+indent_size = 2
```

Comment:
Currently we have `indent_size` with `2` in the repository. Are we thinking in change this? Because this change has a big impact o a lot of other files.